### PR TITLE
Feature issue on englishsite

### PR DESCRIPTION
**Pull request by vijivasanthan**

### CODE REVIEW

**Comment by vijivasanthan** [comment]
[8ATD1: Fix I18N (English-languages) on webapp](https://trello.com/c/w0Jsx7uT/1021-8atd1-fix-i18n-english-languages-on-webapp)

File: app/scripts/filters/avatars.js [hunk 1]
```diff
@@ -91,14 +92,16 @@ define(
       {
         return function (slot)
         {
-          var weeklyslotEndDay = [0,'maandag','dinsdag','woensdag','donderdag','vrijdag','zaterdag','zondag'];
+          var weeklyslotEndDay = [$rootScope.ui.teamup.confirms.day.sunday, $rootScope.ui.teamup.confirms.day.monday,$rootScope.ui.teamup.confirms.day.tuesday,$rootScope.ui.teamup.confirms.day.wednesday,$rootScope.ui.teamup.confirms.day.thursday,$rootScope.ui.teamup.confirms.day.friday,
```
Comment (leonjoosse) [inline app/scripts/filters/avatars.js]: Looks like this array is defined multiple times in the project. Maybe find a way to define it once in the project. If you need to change it in the future, you will only need to replace it in once place.

File: app/scripts/controllers/team-telephone/timeline/agenda-timeline.js [hunk 2]
```diff
@@ -179,6 +181,7 @@ define(
          */
         function slotDatesValid(slot)
         {
+
```
Comment (leonjoosse) [inline app/scripts/controllers/team-telephone/timeline/agenda-timeline.js]: There are quite some new empty lines added throughout the pull request. Please avoid those as it clutters the commits (and also takes longer to review a pull request).

Comment (vijivasanthan) [reply app/scripts/controllers/team-telephone/timeline/agenda-timeline.js]: Okay.got it.will avoid it in future..

**Pull request merged**